tests/functional/api/test_groups.py

```diff
@@ -81,9 +81,11 @@ def test_it_adds_member_to_group(self, app, user, group, auth_client_header):
 
         assert user in group.members
 
-    def test_it_ignores_forwarded_user_header(self, app, user, factories, group, auth_client_header):
+    def test_it_ignores_forwarded_user_header(self, app, user, factories, group, db_session, auth_client_header):
         headers = auth_client_header
         user2 = factories.User()
+        db_session.commit()
```

Comment:
I'm wondering why a commit needed to be added here and whether it could be a `flush()` instead?